workflow/rules/predictions.smk

```diff
@@ -27,15 +34,15 @@ rule generate_e2g_predictions:
 		final_features = os.path.join(RESULTS_DIR, "{biosample}", "genomewide_features.tsv.gz"),
 	params:
 		epsilon = config["epsilon"],
-		feature_table_file = os.path.join(RESULTS_DIR, "{biosample}", "feature_table.tsv"),
-		trained_model = lambda wildcards: get_trained_model(wildcards.biosample),
+		feature_table_file = lambda wildcards: os.path.join(str(BIOSAMPLE_DF.loc[(BIOSAMPLE_DF["biosample"]==wildcards.biosample) & (BIOSAMPLE_DF["model_dir_base"]==wildcards.model_name), "model_dir"]), "feature_table.tsv"),
+		trained_model = lambda wildcards: os.path.join(str(BIOSAMPLE_DF.loc[(BIOSAMPLE_DF["biosample"]==wildcards.biosample) & (BIOSAMPLE_DF["model_dir_base"]==wildcards.model_name), "model_dir"]), "model.pkl"),
```

Comment:
These look really complicated. Are you not able to utilize the existing `get_feature_table_file` function in utils.smk? We should be able to say something like
`feature_table_file = lambda wildcards: get_feature_table_file(wildcards.biosample, wildcards.model_name)`. And perhaps that function will implement what you have here

same goes with getting the model and thresholds